src/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixer.php

```diff
@@ -25,8 +28,21 @@
 use PhpCsFixer\Tokenizer\Token;
 use PhpCsFixer\Tokenizer\Tokens;
 
-final class NoSuperfluousPhpdocTagsFixer extends AbstractFixer
+final class NoSuperfluousPhpdocTagsFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    /** @var bool */
+    private $allowMixed = false;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        $this->allowMixed = $this->configuration['allow_mixed'];
```

Comment:
IMO there is no need for an extra `$allowMixed` property as it simply duplicates `$this->configuration['allow_mixed']`. I would drop it.